### PR TITLE
add header in shell-and-git.md

### PR DESCRIPTION
**Pull request by sebsch1337**
Please review my added header in shell-and-git.md :)

### CODE REVIEW

File: shell-and-git/shell-and-git.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# SHELL AND GIT
+
```
Comment (Mark-Krensel) [inline shell-and-git/shell-and-git.md:2]: comment

**Review by Mark-Krensel — APPROVED** [review]

File: shell-and-git/shell-and-git.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+lorem ipsum ...
```
Comment (kai-franke) [inline shell-and-git/shell-and-git.md:3]: text not creative enough. ;)

File: shell-and-git/shell-and-git.md (hunk 1 above)
Comment (sebsch1337) [reply shell-and-git/shell-and-git.md:2]: nice

File: shell-and-git/shell-and-git.md (hunk 2 above)
Comment (sebsch1337) [reply shell-and-git/shell-and-git.md:3]: shit :O

**Pull request merged**